packages/database/src/migrations/20210602030420-AddFijiCOVIDVaccinatedByFacilityOverlays-modifies-data.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const createMapOverlay = doseNum => ({
+  id: `FJ_COVID_TRACKING_Dose_${doseNum}_Facility`,
+  name: `COVID-19 Vaccine Dose ${doseNum} (Facility)`,
+  userGroup: 'COVID-19',
+  dataElementCode: doseNum === 1 ? 'COVIDVac4' : 'COVIDVac8',
+  measureBuilderConfig: {
+    aggregations: [
+      {
+        type: 'FINAL_EACH_DAY',
+      },
+    ],
+    entityAggregation: {
+      dataSourceEntityType: 'facility',
+    },
+  },
+  measureBuilder: 'sumAllPerOrgUnit',
```

Comment:
This migration is for: beyondessential/tupaia-backlog#2906
The overlay is heatmap at facility level. We want to get **latest for each day** for a facility, and sum all of them to get the total number of doses